processor/k8sprocessor/kube/client.go

```diff
@@ -349,8 +351,8 @@ func (c *WatchClient) shouldIgnorePod(pod *api_v1.Pod) bool {
 		}
 	}
 
-	// Check well known names that should be ignored
-	for _, rexp := range podNameIgnorePatterns {
+	// Check if user requested the pod to be ignored through configuration
+	for _, rexp := range c.Exclude.Regex {
```

Comment:
If we make the changes suggested below, this should read something like the following:

```go
for _, excludedPod := range c.Exclude.Pods {
 		if excludedPod.Name.MatchString(pod.Name) {
```

Reply:
Thanks for the feedback and help! The requested changes have been made.